admin_guide/cluster_management.rst

```diff
@@ -193,6 +193,54 @@ The coordinator node needs to know roles' passwords in order to communicate with
 
 Sometimes workers need to connect to one another, such as during :ref:`repartition joins <repartition_joins>`. Thus each worker node requires a copy of the .pgpass file as well.
 
+.. _phone_home:
+
+Checks For Updates and Cluster Statistics
+#########################################
+
+Unless you opt out, Citus checks if there is a newer version of itself during installation and every twenty-four hours thereafter. If a new version is available, Citus emits a notice to the database logs.
+
+During the check for updates, Citus also sends general information about the running cluster to Citus Data company servers. This helps us understand how Citus is commonly used and thereby improve the product. As explained below, the reporting is opt-out and does **not** contain personally identifying information about schemas, tables, queries, or data.
+
+What we Collect
+---------------
+
+* Citus checks if there is a newer version of itself, and if so emits a notice to the database logs.
```

Comment:
What is the notice? Should we document this separately in Docs?

Reply:
The only other spots in the docs talking about logging are a section in Cloud, multi task log level in configuration, and in our imported articles.

I can add a little example of the log message right in this section.

Reply:
(added example log message in intro section above)